Vagrantfile

```diff
@@ -0,0 +1,16 @@
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.provision :shell, path: "scripts/start-services.sh"
+  config.vm.network :forwarded_port, host: 3500, guest: 3500 # Thimble
+  config.vm.network :forwarded_port, host: 5432, guest: 5432 # Postgres
+  config.vm.network :forwarded_port, host: 2015, guest: 2015 # publish.webmaker.org
+  config.vm.network :forwarded_port, host: 8001, guest: 8001 # Published projects
+  config.vm.network :forwarded_port, host: 1234, guest: 1234 # id.webmaker.org
+  config.vm.network :forwarded_port, host: 3000, guest: 3000 #login.webmaker.org
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 2048
+    v.cpus = 2
```

Comment:
This Vagrant configuration currently fails on Windows, for what appears to be lack of symlink support. Some people have resolved this by adding this configuration option to their Vagrant configuration:

```ruby
vm.customize ["setextradata", :id, "VBoxInternal2/SharedFoldersEnableSymlinksCreate/vagrant", "1"]
```

While others have resolved this issue by using the NPM flag: ```--no-bin-links```, I've tried using both of these methods and didn't manage to get either of them to work. I found a good article of the problem here: http://perrymitchell.net/article/npm-symlinks-through-vagrant-windows/.